frontend/src/components/collective/CollectiveExpenseItem.js

```diff
@@ -39,6 +39,13 @@ const CollectiveExpenseItem = ({
 
   let src, srcSet, href, receiptMessage, receiptHelpTip;
 
+  let submittedByName = user && user.name;
+
+  // show email only if the user can view receipt
+  if (canViewReceipt) {
+    submittedByName = user && user.name && user.email ? `${user.name} - ${user.email}` : `${user.email}`;
```

Comment:
I wonder if we should rather show the PayPal email. The email is not confirmed, but the PayPal email is the one where the money will go to. So might double check that value before approving.